Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1981,7 +1981,7 @@ def coro_noop():
 
         regex = (r'^<CoroWrapper %s\(?\)? .* at %s:%s, .*> '
                     r'was never yielded from\n'
-                 r'Coroutine object created at \(most recent call last\):\n'
+                 r'Coroutine object created at \(most recent call last, maybe truncated\):\n'
```

Comment:
Can we replace "maybe truncated" to "truncated to {} last lines"?

Reply:
"truncated to {DEBUG_STACK_DEPTH} frames" if I understood correctly